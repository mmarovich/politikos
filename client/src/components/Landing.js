import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import '../../styles/Landing.css';

class Landing extends Component {
    render() {
        return (
            <div className="landing-container">
                <div className="introduction-container">
                    <h4>Introducing</h4>
                    <h1>Politikos</h1>
                    <h5>
                        Welcome to Politikos, your resource for tracking and collaborating on your respresentatives.
                        Here you can learn who your representatives are in a federal, state, and local level. You can
                        collaborate on everything that they do and help to educate yourself and others.
                    </h5>
                </div>

                <Link to="/signup"><button className="signup-button">Signup</button></Link>
                <Link to="/login"><button className="login-button">Login</button></Link>
            </div>
        )
    }
}

export default Landing;