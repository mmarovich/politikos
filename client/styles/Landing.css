.landing-container {
    width: 70%;
    min-height: calc(100% - 94px - 56px);
    margin: 30px auto 0;
    text-align: center;
}

.introduction-container {
    margin: 30px;
}

.introduction-container > h1 {
    font-family: 'Cinzel', serif;
}

h1, h4, h5 {
    padding: 2px;
}