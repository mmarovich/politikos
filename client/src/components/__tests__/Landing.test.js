import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router'
import Landing from '../Landing'

describe('Landing component', () => {
    it('render Landing', () => {

        const tree = renderer.create(
            <MemoryRouter>
                <Landing />
            </MemoryRouter>
        ).toJSON()

        expect(tree).toMatchSnapshot()
    })
})