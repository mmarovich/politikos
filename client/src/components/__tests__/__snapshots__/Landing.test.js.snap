exports[`Landing component render Landing 1`] = `
<div
  className="landing-container">
  <div
    className="introduction-container">
    <h4>
      Introducing
    </h4>
    <h1>
      Politikos
    </h1>
    <h5>
      Welcome to Politikos, your resource for tracking and collaborating on your respresentatives. Here you can learn who your representatives are in a federal, state, and local level. You can collaborate on everything that they do and help to educate yourself and others.
    </h5>
  </div>
  <a
    href="/signup"
    onClick={[Function]}>
    <button
      className="signup-button">
      Signup
    </button>
  </a>
  <a
    href="/login"
    onClick={[Function]}>
    <button
      className="login-button">
      Login
    </button>
  </a>
</div>
`;
